packages/terra-core-docs/src/terra-dev-site/doc/pills/example/DisclosurePill.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useRef, useState } from 'react';
+import uniqueid from 'lodash.uniqueid';
+import Popup from 'terra-popup';
+import PillList, { Pill } from 'terra-pills';
+import Spacer from 'terra-spacer';
+
+const DisclosurePill = () => {
+  const [open, setOpen] = useState(false);
+  const pillRef = useRef();
+  const disclosurePillId = `terra-pills-example-disclosure-pill-${uniqueid()}`;
+  const disclosurePopupId = `terra-pills-example-disclosure-popup-${uniqueid()}`;
+
+  const setPillNode = (node) => {
+    pillRef.current = node;
+  };
+
+  const getPillNode = () => pillRef.current;
+
+  const handleOnSelect = () => {
+    setOpen(true);
+  };
+
+  const handleOnRequestClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <PillList
+      ariaLabel="Example of a Basic Pill with a Popup"
+    >
+      <Pill
```

Comment:
Consider writing these examples to include multiple Pills in the PillList. I think that would result in implementations that are more representative of consumer usage.

Reply:
At a minimum we should have more examples of pill list